tests/resources/functions/go/latest/tests.go

```diff
@@ -74,16 +74,16 @@ When you can have two!
 			cookie = "missing"
 		}
 
-		return Context.Res.Text("OK", 200, map[string]string{
+		return Context.Res.Text("OK", Context.Res.WithHeaders(map[string]string{
```

Comment:
maybe im missing something here, but this naming convention doesn't look right `WithHeaders`

Reply:
Same as above applies https://github.com/open-runtimes/open-runtimes/pull/284#discussion_r1686212519